### DISCUSSION TITLE
unable to use nextjs app for frontend. How to integrate nextjs with wails app?

### DISCUSSION THREAD

**Discussion by learncodingforweb** [wailsapp/wails#2737] [Q&A]
### Is your feature request related to a problem? Please describe.

**How to integrate nextjs with wails app?**

### Describe the solution you'd like

There should way to nextjs based ui for desktop app development using wails 

### Describe alternatives you've considered

Currently using reactjs for development

**Comment by mooijtech** [comment]
See https://github.com/LGiki/wails-template-nextjs

Note: Live reload is broken for me, you have to have Inspect Element network tab open with disabled cache then reload.

**Comment by learncodingforweb** [comment]
create issue using nextjs using App Router ** nextjs with app router is not working # 4 ** on [https://github.com/LGiki/wails-template-nextjs](url)
